contracts/voting-verifier/src/execute.rs

```diff
@@ -259,7 +259,7 @@ fn create_worker_set_poll(
 ) -> Result<PollId, ContractError> {
     let id = POLL_ID.incr(store)?;
 
-    let poll = WeightedPoll::new(id, snapshot, block_height + expiry, 1);
+    let poll = WeightedPoll::new(id, snapshot, block_height.saturating_add(expiry), 1);
```

Comment:
same here

Reply:
I kind of feel the same way as here: https://github.com/axelarnetwork/axelar-amplifier/pull/296#discussion_r1525694763

Reply:
see my response there